src/cmds/changelogs/single.js

```diff
@@ -45,7 +45,7 @@ module.exports = class SingleChangelogCommand {
       return Promise.reject(new Error(`No file path provided. Usage \`${config.get('cli')} ${this.usage}\`.`));
     }
 
-    if (filePath.endsWith('.md') === false || !filePath.endsWith('.markdown') === false) {
+    if (!(filePath.endsWith('.md') || filePath.endsWith('.markdown'))) {
       return Promise.reject(new Error('The file path specified is not a markdown file.'));
```

Comment:
Does this need to be capitalized?

```suggestion
      return Promise.reject(new Error('The file path specified is not a Markdown file.'));
```